src/goInstallTools.ts

```diff
@@ -183,6 +195,8 @@ export function installAllTools(updateExistingToolsOnly: boolean = false) {
 		'golangci-lint': 'Linter)',
 		'revive': '\t\t(Linter)',
 		'go-langserver': '(Language Server)',
+		'bingo': '(Language Server)',
+		'golsp': '(Language Server)',
```

Comment:
To help users differentiate lets add some suffixes here like

- Language Server by Sourcegraph
- Language Server by Google
- Language Server by saibing

Reply:
Done